instrumentation/net/http/otelhttp/handler_test.go

```diff
@@ -82,7 +82,19 @@ func TestHandlerBasics(t *testing.T) {
 		attribute.String("test", "attribute"),
 	}
 
-	assertMetricAttributes(t, attributesToVerify, meterimpl.MeasurementBatches)
+	var statusCodeBatch oteltest.Batch
+	var measurementBatches []oteltest.Batch
+	for _, batch := range meterimpl.MeasurementBatches {
+		batchName := batch.Measurements[0].Instrument.Descriptor().Name()
+		if batchName == RequestCount {
+			statusCodeBatch = batch
+			continue
```

Comment:
is this continue needed? 

If I am not missing anything then I think that we could `assertMetricAttributes` for `RequestCount` metric as well

Reply:
Thanks, I will check it. Maybe I missed something.